privatenetworks.go

```diff
@@ -0,0 +1,64 @@
+package scalingo
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/errgo.v1"
+
+	httpclient "github.com/Scalingo/go-scalingo/v8/http"
+	"github.com/Scalingo/go-utils/errors/v2"
+	"github.com/Scalingo/go-utils/logger"
+	"github.com/Scalingo/go-utils/pagination"
+)
+
+type PrivateNetworksService interface {
+	PrivateNetworksDomainsList(ctx context.Context, app string, page uint, perPage uint) (pagination.Paginated[[]PrivateNetworkDomain], error)
+}
+
+var _ PrivateNetworksService = (*Client)(nil)
+
+type PrivateNetworkDomain = string
```

Comment:
nitpick: the equal isn't required, and I would say most of the time not used.
```suggestion
type PrivateNetworkDomain string
```